src/onnxruntime.cc

```diff
@@ -2674,6 +2674,10 @@ TRITONBACKEND_ModelFinalize(TRITONBACKEND_Model* model)
 TRITONBACKEND_ISPEC TRITONSERVER_Error*
 TRITONBACKEND_ModelInstanceInitialize(TRITONBACKEND_ModelInstance* instance)
 {
+  // NOTE: If the corresponding TRITONBACKEND_BackendAttribute is enabled by the
```

Comment:
@tanmayv25 added note here as requested.

Also added some generic BackendAttribute docs in the backend repo here: https://github.com/triton-inference-server/backend/pull/87 - please review as well.